proto/tendermint/services/block/v1/block_service.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package tendermint.services.block.v1;
+
+option go_package = "github.com/cometbft/cometbft/proto/tendermint/services/block/v1";
+
+import "tendermint/services/block/v1/block.proto";
+
+// BlockService provides information about blocks
+service BlockService {
+  // GetBlock retrieves the block information at a particular height.
+  rpc GetByHeight(GetByHeightRequest) returns (GetByHeightResponse);
+
+  // GetLatest retrieves the latest block.
+  rpc GetLatest(GetLatestRequest) returns (GetLatestResponse);
+
+  // GetLatestHeight returns a stream of the latest block heights committed by
+  // the network. This is a long-lived stream that is only terminated by the
+  // server if an error occurs. The caller is expected to handle such
+  // disconnections and automatically reconnect.
+  rpc GetLatestHeight(GetLatestHeightRequest) returns (stream GetLatestHeightResponse);
```

Comment:
Errors also locally:
```
$ make proto-lint 
Linting Protobuf files
proto/tendermint/services/block/v1/block_service.proto:20:3:RPC "GetLatestHeight" is server streaming.
exit status 100
make: *** [proto-lint] Error 1
```

In `main` the same `make` command works, and the files are identical.